australian-guide-to-legal-citation-3rd-edition.csl

```diff
@@ -0,0 +1,587 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Australian Guide to Legal Citation</title>
+    <title-short>AGLC</title-short>
+    <id>http://www.zotero.org/styles/australian-guide-to-legal-citation-3rd-edition</id>
+    <link href="http://www.zotero.org/styles/australian-guide-to-legal-citation-3rd-edition" rel="self"/>
+    <link href="https://forums.zotero.org/discussion/4841/new-australian-legal-citation-style/?Focus=20831#Comment_20831" rel="documentation"/>
```

Comment:
This documentation link doesn't highlight a comment for me. Seems to be the old forums syntax. (@adam3smith, you're currently reusing this old link for the 4th edition)